tests/Microsoft.Identity.Test.Unit/BrokerTests/WamTests.cs

```diff
@@ -634,7 +634,7 @@ public async Task ATI_WithoutPicker_AccountMatch_Organizations_Async()
                     "https://login.microsoft.com", "organizations");
                 var webAccount = new WebAccount(wamAccountProvider, "user@contoso.com", WebAccountState.Connected);
                 var webTokenRequest = new WebTokenRequest(wamAccountProvider);
-
+                _webAccountProviderFactory.IsOrganizationsProvider(wamAccountProvider).Returns(true);
```

Comment:
See my comment in WamBroker, do we need a test where this would return false?

Reply:
I think there are tests like this, but let me add another one.